docs/workforce-management-framework/topics/con-security-introduction.adoc

```diff
@@ -0,0 +1,50 @@
+[]
+= Introduction to {WFM-RC-NameShort} security features
+
+Developers can choose one of the following authentication and authorisation providers
+link:http://www.keycloak.org/[Keycloak] and link:http://passportjs.org/[Passport.js].
+{WFM-RC-NameShort} is build around security interface, which abstracts from any security libraries.
```

Comment:
Once we extract securiyy module you can make this clear that both of them implement this particular interface in the cloud. We need to implicitly say that we talking here only about server side support.